crates/sui-indexer-alt-graphql/src/args.rs

```diff
@@ -11,6 +11,15 @@ use url::Url;
 
 use crate::RpcArgs;
 
+#[derive(clap::Args, Default, Clone, Debug)]
+pub struct TxExecFullNodeArgs {
```

Comment:
I think we can just simplify this to `FullnodeArgs`, because we will hopefully be using this for more things in future.